packet/SpotMarketRequest.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+# SPDX-License-Identifier: LGPL-3.0-only
+
+
+class SpotMarketRequest:
+    def __init__(self, data):
+        self.id = data.get("id")
+        self.created_at = data.get("created_at")
+        self.devices = data.get("devices")
+        self.devices_max = data.get("devices_max")
+        self.devices_min = data.get("devices_min")
+        self.end_at = data.get("end_at")
+        self.facilities = data.get("facilities")
+        self.max_bid_price = data.get("max_bid_price")
+        self.plan = data.get("plan")
```

Comment:
Can this be converted to a `Plan()` safely? I have the same question for `Project()`.